app/src/main/java/org/openobservatory/ooniprobe/common/service/ServiceUtil.java

```diff
@@ -101,7 +101,7 @@ public static void callCheckInAPI(Application app) {
         if (suite != null) {
             Intent serviceIntent = new Intent(app, RunTestService.class);
             serviceIntent.putExtra("testSuites", testSuites);
-            serviceIntent.putExtra("storeDB", true);
+            serviceIntent.putExtra("storeDB", false);
```

Comment:
What does this change do?

Reply:
This change is linked to the earlier PR https://github.com/ooni/probe-android/pull/543 which changed the value. This corrects that change which was not supposed to be merged.